test_programs/execution_success/field_or_integer_static_trait_method/src/main.nr

```diff
@@ -0,0 +1,25 @@
+trait Read {
```

Comment:
Can this test be moved to compile_success_empty?

Reply:
It has a couple of `assert_eq`. I could change the code to run at comptime instead. Would that be okay?

Reply:
assert_eq should still be optimized out as long as it doesn't use program inputs

Reply:
Nice, I didn't know that